src/main/java/io/vertx/core/spi/metrics/NetworkMetrics.java

```diff
@@ -20,7 +20,12 @@
 
 /**
  * An SPI used internally by Vert.x to gather metrics on a net socket which serves
- * as a base class for TCP or UDP.
+ * as a base class for TCP or UDP.<p/>
+ *
+ * All the methods on this object including the methods inherited from the super interfaces are invoked
+ * with {@link io.vertx.core.Context} and thread of the network server/client and therefore are the same than
```

Comment:
well actually I meant that for a given request/connection that thread will be the same as Netty assigns a Netty event loop (which can be Vert.x worker or not) to a channel handler.